features/machine/OCP24363.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Machine taints
+
+  # @author miyadav@redhat.com
+  # @case_id OCP-24363
+  @admin
+  Scenario: [MAO] Reconciling machine taints with nodes
+    Given I have an IPI deployment
+    Then I store the last provisioned machine in the :machine clipboard
+
+   Then I run the :get admin command with:
+      | resource      | machine                 |
+      | resource_name | <%= cb.machine %>       |
+      | o             | yaml                    |
+   And I save the output to file>machines_to_taint.yml
+
+   Given I have the machine yaml file machines_to_taint.yml and I merge it with taints template URL: <%= ENV['BUSHSLICER_HOME'] %>/testdata/cloud/<path>/<file>
```

Comment:
I can't completely understand here. Looks like you only want to add taints. But `machinetaint_template.yml` is a full `Machine`.
I think that to make it readable we need to have it like:
```
Given I add the following taints to machine "<%= cb.machine %>":
"""
     - effect: 
       key: 
       value:     
"""

Then the step do everything needed. You will not need a separate test file as well.

Reply:
@akostadinov  , I was unable to make use of above , as used patch command instead ,  some other steps are pending , thanks for your help in looking into this and enabling me to use better approaches